src/app/screens/BookDetail/components/BookAttr/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import defaultImage from '../../../../../assets/ASSETS/default_book.svg';
+import './styles.css';
+
+function BookAttr({ book }) {
+  return (
+    <div className="props-container-detail">
+      {book.image_url ? (
+        <img className="detail-image" src={book.image_url} alt={book.title} />
+      ) : (
+        <div className="detail-image">
+          <img src={defaultImage} className="detail-image-default" alt={book.title} />
+        </div>
+      )}
+      <div className="props-texts-container">
+        <h1 className="detail-title">{book.title}</h1>
+        <h2 className="detail-author detail-sub-headears">{book.author}</h2>
+        <h2 className="detail-year detail-sub-headears">{book.year}</h2>
+        <h2 className="detail-genre detail-sub-headears">{book.genre}</h2>
+        <h3 className="detail-description">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
+          et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat.
+        </h3>
+        <button className="detail-rent">Alquilar</button>
```

Comment:
create a `strings.js` file in `/BookDetail` and move `Alquilar` and `Lorem...`. 

and import the strings from the file